supported-targets.md

```diff
@@ -47,10 +57,12 @@
 | Microchip         | ATSAM D5/E5 | ✔ Full        | v1.6.2             | samx5x.c       | -
 | Atmel             | ATSAM D     | ✔ Full        | v1.6               | samd.c         | -
 | Atmel             | ATSAM 4L    | ✔ Full        | v1.6               | sam4l.c        | -
+| Atmel             | ATSAMC 21   | ✔ Full        | v2.0.0             | samd.c         | -
```

Comment:
going by the lines above/below and to put it in alphabetical order, we suggest this should be `ATSAM C21` and pushed up two lines ot just above the `ATSAM D` line.